.prettierignore

```diff
@@ -11,13 +9,6 @@ packages/turf-directional-mean/test/in/bus_route_utm.json
 # is actually output
 packages/turf/turf.min.js
 
-# Ignore test fixture json in case intentional line breaks help with coord
-# readability.
-packages/turf-*/test/in/**
-packages/turf-*/test/out/**
-packages/turf-*/test/true/**
-packages/turf-*/test/false/**
```

Comment:
We were actually intentionally ignoring the json/geojson files in the test directories I believe for whitespace reasons. I changed prettier to target longer lines which is maybe a better compromise with readability and gets us back to being standardized.